ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -0,0 +1,529 @@
+
+// @ts-ignore
+const languageToggle = e => {
+  e.stopPropagation();
+  const elements = document.querySelectorAll('.panel, .toggle');
+  for (let i = 0; i < elements.length; ++i) {
+    const classNames = elements[i].className.split(' ');
+    const indexOfHide = classNames.indexOf('hide');
+    if (indexOfHide !== -1) {
+      classNames.splice(indexOfHide, 1);
+    } else {
+      classNames.push('hide');
+    }
+    elements[i].className = classNames.join(' ');
+  }
+};
+
+const pageTitle = 'News in your language - BBC News';
+const pageDescription = 'A list of BBC World Service language services';
+const lang = 'en';
+
+const atiAnalytics = {
+  campaigns: null,
+  categoryName: null,
+  contentType: STATIC_PAGE,
+  language: lang,
+  ldpThingIds: null,
+  ldpThingLabels: null,
+  pageIdentifier: 'ws.languages.page',
+  pageTitle,
+  producerId: null,
+  producerName: 'DEFAULT',
+};
+
+const pageLayout = () => {
+  return (
+    <>
+      <ATIAnalytics atiData={atiAnalytics} />
+      <ChartbeatAnalytics title={pageTitle} />
+      <MetadataContainer
+        title={pageTitle}
+        lang={lang}
+        description={pageDescription}
+        openGraphType="website"
+        hasAmpPage={false}
+      />
+      <style>{morphCSS1}</style>
+      <style>{morphCSS2}</style>
```

Comment:
I'd wrap these in `Head` from `next/head`, like:
```      
<Head>
        <style>{morphCSS1}</style>
        <style>{morphCSS2}</style>
</Head>
```

Basically the same as `react-helmet`, but Nexts native implementation.

Reply:
`Head` didn't work, so I put in `Helmet`

Reply:
Hmm. Seemed fine when I tried it locally. Helmet is ok for now but if the Next one doesn't work we'll need to figure out why as we want to migrate off of using react-helmet.

Reply:
yeah - I pulled in using `next/document` rather than `next/head` - I've changed this so it now works 🎉